dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -225,27 +233,57 @@ def SignalHandler(*unused_argvs):
   sys.exit(0)
 
 
+def SetupLogging():
+  """Sets up a logging handler with dftimewolf's custom formatter."""
+  # Clear root handlers (Turbinia is setting them)
+  root_log = logging.getLogger()
+  root_log.handlers = []
+
+  # Add and silence a default stream handler, this is automatically set
+  # when other libraries call logging.info() or similar methods.
+  root_handler = logging.StreamHandler()
+  root_handler.addFilter(lambda x: False)
+  root_log.addHandler(root_handler)
+
+  # We want all error messages. Maybe make this customizable in the future?
```

Comment:
technically you are not gettingn all `error` messages, but DEBUG and above... you're getting all messages. and maybe add a TODO here to have this configurable.

There could be a default value of INFO and a `--verbose`, `--debub` parameter to the tool tha twoul dchange it from INFO level to DEBUG

Reply:
Done.